source/features/image-alternatives.js

```diff
@@ -0,0 +1,25 @@
+import {h} from 'dom-chef';
+import {safeElementReady} from '../libs/utils';
+
+export default async () => {
+	const imgContainers = document.querySelectorAll('.AdaptiveMedia > div > div > div');
```

Comment:
This selector is very fragile to changes. I would try to be more specific and use less `>`.

Reply:
👍